chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -125,9 +115,14 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
         const globalDir = props.controlProps?.dir ?? getLocaleDirection(props.chatConfig?.ChatWidgetLanguage?.msdyn_localeid);
         dispatch({ type: LiveChatWidgetActionType.SET_GLOBAL_DIR, payload: globalDir });
 
-        if (state.domainStates?.liveChatContext) {
-            const optionalParams = { liveChatContext: state.domainStates?.liveChatContext };
-            initStartChat(chatSDK, dispatch, setAdapter, optionalParams);
+        //Check if auth settings enabled, do not connect to existing chat from cache during refresh/re-load
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const isAuthenticationSettingsEnabled = (props.chatConfig?.LiveChatConfigAuthSettings as any)?.msdyn_javascriptclientfunction ? true : false;
+        if (!isAuthenticationSettingsEnabled) {
+            if (state.domainStates?.liveChatContext && state.appStates.conversationState === ConversationState.Active) {
```

Comment:
`state.domainStates?.liveChatContext` would return true if the value was `{}`

I'd suggest to do `state.domainStates?.liveChatContext && Object.keys(state.domainStates?.liveChatContext).length` to ensure `state.domainStates?.liveChatContext` has properties.

Reply:
Created an util function for this.